src/docs/reference/scale-to-zero.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Scale to Zero
+---
+
+Scale to Zero means reducing unnecessary costs by ensuring you only use what you need, when you need it.
+
+## What is App Sleeping?
+
+App sleeping enables automated detection of an inactive [service](/develop/services) based on outbound traffic.
+
+Enabling App Sleep on a service tells Railway to stop the service when it is inactive, effectively reducing the overall cost to run it.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1696017787/events_gjkaob.png"
+alt="App Sleeping in the Activity Feed"
+layout="intrinsic"
+width={700} height={460} quality={100} />
+
+### Inactive Service Detection
```

Comment:
Thoughts on making "inactive" / "active" into "slept" and "wake" for consistency? This feels like it may be semantically referring to the same thing using different terms